website/templates/project/modal_add_pointer.mako

```diff
@@ -37,14 +37,14 @@
                         <table class="table table-striped">
                             <tbody data-bind="foreach:{data:results, afterRender:addTips}">
                                 <tr data-bind="if:!($root.selected($data))">
-                                    <td>
+                                    <td style="width: 50px">
```

Comment:
Right; I meant you can add your own class rather than repeating this inline style. Something like `osf-link-td`.